README.adoc

```diff
@@ -1,19 +1,32 @@
-= https://github.com/input-output-hk/plutus[Plutus Platform]
+= https://github.com/input-output-hk/plutus[The Plutus Platform and Marlowe]
 :email: plutus@iohk.io
 :author: Input Output HK Limited
 :toc: left
 :reproducible:
 
-The Plutus Platform enables you to:
+The Plutus Platform is an application development platform for developing distributed applications using the Cardano blockchain; and Marlowe is a platform specifically for financial products, built on top of Plutus.
```

Comment:
I'm not sure if it's helpful to provide context for the name "Plutus"; i.e. something like:

"Plutus is the scripting language for the Cardano ledger, and the Plutus Platform is an application development environment for building distributed applications on the Cardano blockchain".

(just because if I was to read this, I immediately wonder "What is Plutus?")

(Also, are the words "blockchain" and "ledger" interchangable? If so, maybe just use one of them?)

Reply:
Hm yes, we have a perpetual issue with "what does 'Plutus' unqualified mean?", so yeah, it would be good to clear that up.

Blockchain and ledger are not totally interchangeable, we talk about that a bit in some of the other docs.